satellite/gracefulexit/endpoint_test.go

```diff
@@ -799,6 +799,40 @@ func TestUpdatePointerFailure_DuplicatedNodeID(t *testing.T) {
 	})
 }
 
+func TestExitDisabled(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount:   1,
+		StorageNodeCount: 2,
+		UplinkCount:      1,
+		Reconfigure: testplanet.Reconfigure{
+			Satellite: func(log *zap.Logger, index int, config *satellite.Config) {
+				config.GracefulExit.Enabled = false
+			},
+		},
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		satellite := planet.Satellites[0]
+		exitingNode := planet.StorageNodes[0]
+
+		require.Nil(t, satellite.GracefulExit.Chore)
+		require.Nil(t, satellite.GracefulExit.Endpoint)
+
+		conn, err := exitingNode.Dialer.DialAddressID(ctx, satellite.Addr(), satellite.Identity.ID)
+		require.NoError(t, err)
+		defer func() {
```

Comment:
Use `defer ctx.Check(conn.Close)`, currently nothing is verifying that `conn.Close` is returning `nil`.